### ISSUE TITLE
Accessibility issue: Unable to move to next workflow step using just keyboard

### ISSUE THREAD

**Issue by StephanieVUGSC** [TexasDigitalLibrary/Vireo#863]
ACCESSIBILITY:   It does not appear to be possible to interact with the submission step   navigation using only keyboard input

**Comment by jcreel** [comment]
Make the continue button and the steps listed at the top respond to Enter keystroke.

**Comment by kaladay** [comment]
Resolved in #862.

**Comment by StephanieVUGSC** [comment]
I have confirmed this fix. Approved.
